vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -43,10 +42,6 @@ def textual_hostname
     {:label => "Hostname", :value => @ems.hostname}
   end
 
-  def textual_ipaddress
-    {:label => "IP Address", :value => @ems.ipaddress}
```

Comment:
@moolitayer I preferred to have the visibility of field that could have a value. Anyway since we're not using it (and we don't plan to use it) we can remove it.